server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/OperationClientInterface.js

```diff
@@ -330,8 +331,29 @@ class OperationClientInterface extends layerProtocol {
             }
         });
     }
+}
 
-
-
+/**
+ * @description This function returns the remote address configured .
+ * @param {String} remoteAddress : remote address of the tcp client .
+ * @returns {promise} string {remoteAddress}
+ **/
+function getConfiguredRemoteAddress(remoteAddress) {
+    return new Promise(async function (resolve, reject) {
+        try {
+            if ("domain-name" in remoteAddress) {
```

Comment:
Please remove the hardcoded values. Instead , declare them as a constant in the module applicationPattern\onfModel\constants\OnfAttributes.js and utilize the declared constant. In this way , we can utilize the attribute name globally across many modules.